packages/mdc-chips/README.md

```diff
@@ -365,7 +365,7 @@ Method Signature | Description
 `removeClassFromLeadingIcon(className: string) => void` | Removes a class from the leading icon element
 `eventTargetHasClass(target: EventTarget, className: string) => boolean` | Returns true if target has className, false otherwise
 `notifyInteraction() => void` | Notifies the Chip Set that the chip has been interacted with\*
-`notifySelection(selected) => void` | Notifies the Chip Set that the chip has been selected or deselected\*\*
+`notifySelection(selected: boolean, shouldIgnore: boolean) => void` | Notifies the Chip Set that the chip has been selected or deselected\*\*
```

Comment:
Let's document second boolean parameter. same for `handleChipSelection`.

Reply:
Done